.github/workflows/mlkem-hax.yml

```diff
@@ -82,11 +82,36 @@ jobs:
       - uses: hacspec/hax-actions@main
         with:
           hax_reference: ${{ github.event.inputs.hax_rev || 'main' }}
-          fstar: v2025.01.17
+          fstar: v2025.02.17
 
       - name: 🏃 Lax ML-KEM crate
         working-directory: libcrux-ml-kem
         run: ./hax.py prove --admit
+
+  prove:
+    runs-on: self-hosted
+    if: ${{ github.event_name == 'merge_group' || github.event_name == 'workflow_dispatch' }}
+    needs:
+      - extract
+      - diff
+      - lax
```

Comment:
We don't actually run these other jobs on the merge queue so we could skip them here.